app/worker.ts

```diff
@@ -262,3 +307,7 @@ self.addEventListener("message", async (event: { data: any }) => {
     data: "OK",
   });
 });
+
+(async () => {
+  aiMaskClient = await AIMaskClient.getWorkerClient();
```

Comment:
See below, could this just trigger on some `initialization` event, then send a `initializationComplete` once this `await` returns or it times out?